<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TicTacToeWindow</class>
 <widget class="QMainWindow" name="TicTacToeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TicTacToeWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>Player vs Player Tic Tac Toe</string>
    </property>
    <widget class="QPushButton" name="board00">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>51</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="board01">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>51</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="board02">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>51</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="board11">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>110</y>
       <width>51</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="board10">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>51</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="board12">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>110</y>
       <width>51</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="board21">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>170</y>
       <width>51</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="board20">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>51</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="board22">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>170</y>
       <width>51</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="IBStatusLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>321</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Reset_Button">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>130</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
